### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by ErikStoupignan**
# Leaderboard: final touches
##  General requirements

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config). ✅ 
- Make sure that you used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). ✅ 
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). ✅ 

## Project requirements

- At this point, you should have a fully working app that uses only basic styles to make the layout work, according to this wireframe:  ✅
image

- Now you should improve the look and feel of the application, adding the styles of your choice. ✅ 
- Please keep the general layout of the wireframe, this is the only mandatory requirement. ✅ 
- You can use plain CSS or any CSS framework, it's up to you. ✅ 

## Screenshot for my project
image

Dear Code reviewer, in my project I added a secret feature. Do you think you can find it? 🚀

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,15 +5,17 @@ import addElement from './modules/addElement.js';
 import getBoard from './modules/Get-Board.js';
 
 const id = 'QgHcOIXPyYQqqLJkTuJF';
+// const id = 'QgHcOIXPyYQqqLJkTuJ1'; //false Id
```
Comment (rustam-tolipov) [inline src/index.js]: - [x] Kindly remove commented code from your file.

File: src/styles/style.css [hunk 2]
```diff
@@ -69,59 +118,230 @@ body {
...
+.ul-score li:nth-child(odd){
+  background-color: rgba(255, 255, 255, 0.54);
+  color: #000;
 }
 
 /* table right */
+.container-addYourScore {
+  /* background-image: url(/assets/add-your-score.jpg);
+  background-size: 150% 150%;
+  background-repeat: no-repeat; */
```
Comment (rustam-tolipov) [inline src/styles/style.css]: - [x] You should make your code as clean as possible. Please remove commented code from your project!

File: src/index.js [hunk 3]
```diff
@@ -5,15 +5,17 @@ import addElement from './modules/addElement.js';
 import getBoard from './modules/Get-Board.js';
```
Comment (rustam-tolipov) [inline src/index.js]: - [x] Why did you left open your json result on console?


Screen Shot 2022-08-26 at 06 02 16

To make your work professional, you shouldn't display any line of code in console with console.log.

File: src/styles/style.css [hunk 4]
```diff
@@ -5,56 +5,105 @@
...
-button {
-  box-shadow: 2px 2px 2px black;
-}
-
 button:hover {
   cursor: pointer;
-  transform: scale(0.9);
 }
 
 body {
```
Comment (rustam-tolipov) [inline src/styles/style.css:12]: I love the styling that you selected! Great work!

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @ErikStoupignan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (ErikStoupignan) [reply src/index.js]: Good evening and thank you very much for your comments @Rustamxon7 .

In a moment, I will make all the corrections.

File: src/styles/style.css (hunk 2 above)
Comment (ErikStoupignan) [reply src/styles/style.css]: You're absolutely right, I got so excited working to make it look pretty that I forgot to clean up the code.

Thank you so much for the correction.

File: src/index.js (hunk 3 above)
Comment (ErikStoupignan) [reply src/index.js]: Oh, I was using it to keep clear control of the information I was sending to the API and I forgot to remove it.

Sorry for the inconvenience and thank you very much for the comment.

File: src/styles/style.css (hunk 4 above)
Comment (ErikStoupignan) [reply src/styles/style.css:12]: Thank you very much and I'm glad you liked it!

**Review by descholar-ceo — APPROVED** [review]
Hi @ErikStoupignan 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Implemented all requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**